src/lib/getDataFromExternalSources.test.ts

```diff
@@ -53,6 +53,28 @@ describe('getDataFromExternalSources', () => {
         IPFS_GATEWAY_MOCK,
       );
     }).to.not.throw();
+    expect(async () => {
+      await getDataFromExternalSources(
+        [
+          {
+            name: 'LSP3Profile2',
```

Comment:
Why the `2`?

Reply:
To make it non-existent or non matching the key